test/IoC.Test/UnitTest/IoCTest.cs

```diff
@@ -0,0 +1,42 @@
+using Xunit;
+using Xunit.Abstractions;
+
+namespace IoC.Test.UnitTest;
+
+public class IoCTest
+{
+    private readonly InstancePool _sut; //SUT stands for System Under Test.
+
+    public IoCTest(ITestOutputHelper output)
+    {
```

Comment:
What is this for?

Reply:
Oh, my bad. At this point that part make no sense. I was playing with some logging info and the way to log something is using a method coming from that injected class.